### PR TITLE
The great clownification...

### PR DESCRIPTION
**Pull request by Teapug**
The clowns....

Oh god....

made everyone spawn with clown mask

also, gave the clown their lubegrenade back :)

#714 

## Summary by CodeRabbit

- **New Features**
	- Added a clown mask (`ClothingMaskClown`) to the starting gear for various job roles including Cargo Technician, Quartermaster, Salvage Specialist, and many more.
	- Enhanced starting gear for roles such as Chef, Librarian, and Medical Doctor with the addition of the clown mask.
	- Introduced new items and capabilities for specific roles, including the Mime and Clown.

- **Bug Fixes**
	- Corrected indentation issues in the Security Cadet job configuration.

These updates expand the equipment options available to users across multiple roles, enhancing gameplay experience.

### CODE REVIEW

**Comment by Icepicked** [comment]
make this have a chance to happen on secret :godo:

**Comment by Teapug** [comment]
> Important
> 
> ## Review skipped
> Auto reviews are disabled on base/target branches other than the default branch.
> 
> Please check the settings in the CodeRabbit UI or the `.coderabbit.yaml` file in this repository. To trigger a single review, invoke the `@coderabbitai review` command.
> 
> You can disable this status message by setting the `reviews.review_status` to `false` in the CodeRabbit configuration file.
> 
> Thank you for using CodeRabbit. We offer it for free to the OSS community and would appreciate your support in helping us grow. If you find it useful, would you consider giving us a shout-out on your favorite social media?
> 
> ❤️ Share
> 🪧 Tips

@coderabbitai review my PR anyways. And write a poem

**Review by Piras314 — CHANGES_REQUESTED** [review]
make the clown mask scary

**Comment by Teapug** [comment]
> make the clown mask scary

the clown mask IS already scary

**Comment by Teapug** [comment]
fine ill make a new sprite

ill just make a red clown mask or something lmao

**Comment by Piras314** [comment]
give it joker lipstick and creepy (possibly glowing?!) eyes

**Comment by Piras314** [comment]
oh yeah and make it irremovable

**Comment by Piras314** [comment]
@coderabbitai pull requests being made to the halloween-2024 branch are meant to be humerous or scary

**Comment by Teapug** [comment]
@Piras314  is it "scary" enough now?

**Review by Piras314 — CHANGES_REQUESTED** [review]
edit equipped-MASK-vox.png (and possibly equipped-MASK-hamster.png :godo:) and meta.json in textures/clothing/mask/clown.rsi

**Comment by Piras314** [comment]
@Teapug the meta.json matters because it's literally the copyright data, credit yourself for the sprites or it's not getting merged

also you have like 3 weeks to draw the vox sprite, halloween isn't for a good while.

File: Resources/Prototypes/Roles/Jobs/Civilian/mime.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@
 - type: startingGear
   id: MimeGear
   equipment:
+    mask: ClothingMaskClown
```
Comment (Piras314) [inline Resources/Prototypes/Roles/Jobs/Civilian/mime.yml:21]: not for mime

Comment (Teapug) [reply Resources/Prototypes/Roles/Jobs/Civilian/mime.yml:21]: # why

Comment (Teapug) [reply Resources/Prototypes/Roles/Jobs/Civilian/mime.yml:21]: okay one sec

**Comment by Teapug** [comment]
@Piras314  Pretty sure its done now.

**Comment by Teapug** [comment]
@Piras314  what about making the mask irremovable? how do i do that :P

File: Resources/Prototypes/Roles/Jobs/Civilian/mime.yml [hunk 2]
```diff
@@ -18,6 +18,7 @@
 - type: startingGear
   id: MimeGear
   equipment:
+    #no mask for mime i guess ;-;.
```
Comment (Piras314) [inline Resources/Prototypes/Roles/Jobs/Civilian/mime.yml:21]: you can make the mime mask freaky too if you want

**Review by Piras314 — APPROVED** [review]

File: Resources/Prototypes/Roles/Jobs/Civilian/mime.yml (hunk 2 above)
Comment (Teapug) [reply Resources/Prototypes/Roles/Jobs/Civilian/mime.yml:21]: nah

**Comment by Piras314** [comment]
send an in-game screenshot and this is good for merge

**Comment by Teapug** [comment]
> send an in-game screenshot and this is good for merge

i can test it tomorrow, cant build and launch it on 4 gigs of ram :godo:

**Comment by Teapug** [comment]
@coderabbitai What do you think about the new clownmask textures? write a poem about the new clown mask textures.

**Comment by Piras314** [comment]
I've changed my mind a little bit- make it removeable

**Comment by Aidenkrz** [comment]
I really don't like the idea of giving everyone irremovable clown masks with the ohio accent (What does that have to do with halloween) for a week. Why not just modify the sprite in this pr, and make a separate PR adding the mask as an option in the loadout (add other costumes to loadouts aswell?)

**Review by Aidenkrz — CHANGES_REQUESTED** [review]
Okay now that i'm a maint again
I like the sprite but the rest of these changes make no sense.
- Mask shouldn't be unremoveable
- People shouldnt spawn with it equipped. Put it in loadouts, or wait till I add halloween costumes to the loadouts in my PR
- Why does it add ohio accent, it has nothing to do with halloween or clowns.

**Comment by Piras314** [comment]
webedit moment

**Comment by Teapug** [comment]
> Okay now that i'm a maint again I like the sprite but the rest of these changes make no sense.
> 
> * Mask shouldn't be unremoveable
> * People shouldnt spawn with it equipped. Put it in loadouts, or wait till I add halloween costumes to the loadouts in my PR
> * Why does it add ohio accent, it has nothing to do with halloween or clowns.
> 
> also why the HELL is every single file change a different commit, not that it really matters but just why.

image

**Pull request closed without merging**